incubator/hnc/cmd/manager/main.go

```diff
@@ -135,7 +137,9 @@ func main() {
 	if debugLogs {
 		logLevel = zapcore.DebugLevel
 	}
-	log := zap.New(zap.Level(logLevel), zap.StacktraceLevel(zapcore.PanicLevel))
+	rlog := zap.NewRaw(zap.Level(logLevel), zap.StacktraceLevel(zapcore.PanicLevel))
```

Comment:
I was a bit confused by the name, maybe `rawLog`?

What's the difference between a raw and regular logger? It might be worth a comment about why you're creating it in two different steps (e.g. just update the code, you don't need to answer this question in GitHub).